catalog-info.yaml

```
@@ -49,3 +49,8 @@ spec:
         observablt-robots: {}
         everyone:
           access_level: READ_ONLY
+      schedules:
+        Daily 7.17:
+          branch: "7.17"
+          cronline: "@daily"
```

https://buildkite.com/docs/pipelines/configure/workflows/scheduled-builds#schedule-intervals-predefined-intervals

```suggestion
        Weekly 7.17:
          branch: "7.17"
          cronline: "@weekly"
```